### DISCUSSION TITLE
Change "default" name on Swagger page

### DISCUSSION THREAD

**Discussion by horseinthesky** [fastapi/fastapi#7800] [Questions] [answered]
Hello.
I have found how to change title and description of Swagger page but couldn't find where to change this `default` block name?
```
app = FastAPI(
    title='Bla API',
    description='Bla description',
...

```
![image](https://user-images.githubusercontent.com/19948672/74936179-bada0080-53fa-11ea-8691-7027cf3cc5f8.png)

**Comment by sm-Fifteen** [comment, marked as answer]
That's the route tag for those routes, which is set to "default" if none is specified. You can set one either [per route](https://fastapi.tiangolo.com/tutorial/path-operation-configuration/#tags) or [per router](https://fastapi.tiangolo.com/tutorial/bigger-applications/#include-an-apirouter-with-a-prefix-tags-responses-and-dependencies), it's mainly to help you organize your documentation by grouping API endpoints together.

**Comment by horseinthesky** [comment]
> it's mainly to help you organize your documentation by grouping API endpoints together.

This is exactly why a wanted to change it. Thank a lot!

**Comment by tiangolo** [comment]
Thanks @sm-Fifteen for the help here! :cake: 

Thanks @horseinthesky for reporting back and closing the issue :+1:

**Comment by gouthamgowrishankar** [comment]
default can be replaced by using tags attributes in decorator
@app.get("/", tags=["health Check"])

**Reply by cjdotcom** [reply]
thanks!

**Comment by TaraJura** [comment]
thanks! I
